Dockerfile

```diff
@@ -1,54 +0,0 @@
-# This file was generated with `make docker-files` and should not
```

Comment:
You're probably aware - undo this change.

Reply:
Hmm. make clean deletes this file so what is the expected pattern for building? Do I need to regerenate the dockerfiles and recommit every time?